Userland/Libraries/LibWeb/HTML/MessagePort.h

```diff
@@ -20,7 +20,8 @@ namespace Web::HTML {
 
 // https://html.spec.whatwg.org/multipage/web-messaging.html#message-ports
 class MessagePort final
-    : public RefCounted<MessagePort>
+    : public JS::Cell
+    , public RefCounted<MessagePort>
     , public Weakable<MessagePort>
     , public DOM::EventTarget
     , public Bindings::Wrappable {
```

Comment:
You were correct, I was able to remove the use of JS::Cell Completely. I did need to keep `RefCounted` so that the `Worker` could keep a counted ref to it while it does background operations, does that much seem correct?